plugin/pkg/rqdata/rqdata.go

```diff
@@ -112,10 +116,34 @@ func NewMapping(emptyValue string) *Mapping {
 			}
 			return ""
 		},
+		"response_ips": responseIpsExtractor,
+		"response_ips_with_log": func(state request.Request) string {
+			result := responseIpsExtractor(state)
+			log.Infof("coredns::policy/firewall, Response IPs are: '%s'", result)
+			return result
+		},
```

Comment:
I presume this option would be used for debugging, in which case it may be better to instead add debug logging to `response_ips` enabled by coredns's _debug_ plugin.